ansible/roles/omeka/tasks/main.yml

```diff
@@ -57,6 +53,22 @@
      - Search-By-Metadata-1.2.zip
    when: omeka_exists.stat.exists == false
 
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka/files owner=www-data group=www-data recurse=yes
+   when: omeka_exists.stat.exists == false
+
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka/admin owner=www-data group=www-data recurse=yes
+   when: omeka_exists.stat.exists == false
+
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka/plugins owner=www-data group=www-data recurse=yes
+   when: omeka_exists.stat.exists == false
+
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka/application owner=www-data group=www-data recurse=yes
```

Comment:
Also not sure why the application folder needs to be `www-data` writeable/owned by `www-data`.